### DISCUSSION TITLE
[Feature] show hidden people in 'name search'

### DISCUSSION THREAD

**Discussion by chodthewacko** [immich-app/immich#8688] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

In the people search,
i.e. explore -> view all (people) -> search people,
you can enter a name to only show thumbnails of people with matching names.

Right now it only shows unhidden people.
It would be very useful if it also showed hidden people whose names match.
This will make it far less tedious to unhide someone you accidentally hid.

Right now the only workaround to try to remember a photo that contains this hidden face,
so you can 'show hidden people' on it and unhide someone.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by balvant813** [comment]
Looks like after 500 faces options to see hidden are almost non existent in web browser. What is option to unhide a person (accidently hidden) if there are more than 500 faces? Any Idea?

**Reply by chodthewacko** [reply]
The easiest way is to track down a picture with that person in it.
 then click 'info' to show people, the 'eye' to show hidden people, then click on the hidden person.
That brings you to that person's page of pictures.  You can then go to the menu on the right and select 'unhide person'

**Reply by ja49619** [reply]
> The easiest way is to track down a picture with that person in it. then click 'info' to show people, the 'eye' to show hidden people, then click on the hidden person. That brings you to that person's page of pictures. You can then go to the menu on the right and select 'unhide person'

works well, but too confusing to find

**Comment by bo0tzz** [comment]
This works as requested now
